### DISCUSSION TITLE
No Github releases anymore

### DISCUSSION THREAD

**Discussion by marvin-wtt** [surveyjs/survey-library#9098] [General]
Releases are still tagged but no release is created anymore

**Comment by JaneSjs** [comment]
Hi @marvin-wtt,

Thank you for your inquiry. This issue has already been reported, and we’re aware of it. Currently, we’re focused on our major release, v2.0, but we plan to address the GitHub releases issue in our `survey-library` repository after the release.

Thank you for your patience.

**Comment by JaneSjs** [comment]
Hi,
We fixed the issue. You can access survey-library releases at https://github.com/surveyjs/survey-library/releases.

Let us know if you have any further questions.
